draft-ietf-quic-recovery.md

```diff
@@ -1150,6 +1150,9 @@ This document has no IANA actions.
 We now describe an example implementation of the loss detection mechanisms
 described in {{loss-detection}}.
 
+The pseudocode in this section is licensed as Code Components; see the
```

Comment:
The agreement in number between "The pseudocode" and "Components" is jarring.  I'd go either....
```suggestion
The pseudocode in this section is licensed as a Code Component; see the
```
or
```suggestion
The sections which follow contain pseudocode.  The contents of these sections are
licensed as Code Components; see the
```
Given that it's multiple blocks, I think keeping it plural (as it originally was) might be better, but I agree on finding a word besides "snippets."

Reply:
I'm using "segments", see if that works.